index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Document</title>
+</head>
+<body>
+    <header class="header">
+        <div class="header-container">
+            <nav class="nav-item1"><a class="Mylogo" href="">My Logo</a></nav>
+            <nav class="nav-item2"></nav>
+            <nav class="nav-item3">
+                <img class="toggle" src="images/toggle.png" alt="">
+            </nav>
+            <div id="myNav" class="overlay">
+                <a class="closebtn"> &times; </a>
+                <div class="overlay-content">
+                    <a class="pop-nav" href="#portfolio">Portfolio</a>
+                    <a class="pop-nav" href="#about">About</a>
+                    <a class="pop-nav" href="#contact">Contact</a>
+                </div>
```

Comment:
- [optional] kindly, Use `<ul>` and `<li>` for your navigation list for sematic reason. 👍 👍